src/configs/configs.ts

```diff
@@ -0,0 +1,5 @@
+const API = 'https://api.breinify.com';
+
```

Comment:
I think it'll be a little more readable to have no newline between these constants, since they are kind of related